pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -57,7 +57,8 @@ rec {
     # InstallCheck phase
     , doInstallCheck ? config.doCheckByDefault or false
 
-    , crossConfig ? null
+    , # TODO(@Ericson2314): Make always true and remove
```

Comment:
There is no business having these sorts of TODO comments until it's accepted (via an RFC) that it will be done.